django/contrib/admin/static/admin/css/forms.css

```diff
@@ -264,11 +263,11 @@ body.popup .submit-row {
 .submit-row input {
     height: 35px;
     line-height: 15px;
-    margin: 0 0 5px 5px;
+    margin: 0 5px 5px 0;
 }
 
 .submit-row input.default {
-    margin: 0 0 5px 8px;
+    margin: 0 4px 5px 0;
```

Comment:
Did some digging and apparently the shouting SAVE has been present since a project named django-flat-theme was [merged](https://github.com/django/django/pull/4627) into Django, 7 years ago now. And it was present in the [initial commit](https://github.com/elky/django-flat-theme/blob/9abdb6aed274cb74e724c9e93a1bfbd753cd0bbb/flat/static/flat/css/forms.css#L229) from that project. So... feature..? I guess?

Reply:
FEATURE IT IS THEN!